integration_tests/tests/forseti/controls/scanner-config_validator_gcs_policy.rb

```diff
@@ -60,7 +60,7 @@
   end
 
   # Assert violations exist for Cloud SQL Location policy
-  describe command("mysql -u #{db_user_name} -p#{db_password} --host 127.0.0.1 --database forseti_security --execute \"SELECT COUNT(*) FROM violations V JOIN forseti_security.scanner_index SI ON SI.id = V.scanner_index_id WHERE SI.inventory_index_id = #{@inventory_id} AND V.resource_id = '#{forseti_cloudsql_instance_name}' AND V.violation_type = 'CONFIG_VALIDATOR_VIOLATION' AND V.rule_name = 'sql_location_denylist';\"") do
+  describe command("mysql -u #{db_user_name} -p#{db_password} --host 127.0.0.1 --database forseti_security --execute \"SELECT COUNT(*) FROM violations V JOIN forseti_security.scanner_index SI ON SI.id = V.scanner_index_id WHERE SI.inventory_index_id = #{@inventory_id} AND V.resource_id = '#{forseti_cloudsql_instance_name}' AND V.violation_type = CONCAT('CV_', V.rule_name);\"") do
```

Comment:
The other assert on line 69 will also need the same update.

Reply:
Done.